### ISSUE TITLE
Support for TransposeConv+ Batchnorm

### ISSUE THREAD

**Issue by anilsathyan7** [ZFTurbo/Keras-inference-time-optimizer#19]
It seems TransposeConv  + BN is not currently supported !!!
Any plans to support this combination?? 
(Anyway this doesn't seem to be difficult to implement ...)

**Comment by ZFTurbo** [comment]
Can you point on TransposeConv layer in Keras code or documentation?

**Comment by anilsathyan7** [comment]
1. https://github.com/keras-team/keras/blob/master/keras/layers/convolutional.py#L628
2. https://keras.io/layers/convolutional/
3. https://github.com/tensorflow/examples/blob/master/tensorflow_examples/models/pix2pix/pix2pix.py#L220
4. https://github.com/jocicmarko/ultrasound-nerve-segmentation/blob/master/train.py

**Comment by ZFTurbo** [comment]
Thanks, I will look into it.

**Comment by ZFTurbo** [comment]
I added support for `Conv2DTranspose` layer. Currently it available in repo. I'll make pypi release a little bit later,
